src/main/java/org/jabref/gui/libraryproperties/LibraryPropertiesDialogViewModel.java

```diff
@@ -61,8 +62,9 @@ public LibraryPropertiesDialogViewModel(BasePanel panel, DialogService dialogSer
         Optional<String> fileDI = metaData.getUserFileDirectory(preferencesService.getUser());
         fileDI.ifPresent(userSpecificFileDirectoryProperty::setValue);
 
-        Optional<String> fileDL = metaData.getLaTexFileDirectory(preferencesService.getUser());
-        fileDL.ifPresent(laTexFileDirectoryProperty::setValue);
+        Optional<Path> fileDL = metaData.getLaTexFileDirectory(preferencesService.getUser());
+        Optional<String> laTexFile = Optional.ofNullable(fileDL.toString());
```

Comment:
This does not work as intended: `fileDL` is an optional and thus `toString` returns something like `Optional[value]`.
Instead, you can use 
```
metadata.getLatexFileDirectory(...)
        .map(Path::toString)
        .ifPresent(laTexFileDirectoryProperty::setValue)
```